css/index.css

```diff
@@ -344,38 +325,15 @@ footer {
      }
 
      /* Main Content Styles */
-     .main-content {
-         display: flex;
-         flex-direction: column;
-         width: 90%;
-     }
-
      .main-content .top-content {
         flex-wrap: wrap;
-        margin-bottom: 0 auto;
-        padding: 8% 0;
-        font-size: 1.6rem;
-        line-height: 1.8rem;
-    }
-
-    .main-content .top-content .text-content {
-        margin-bottom: 1.8rem;
-    }
-
-    .main-content .middle-img {
-        width: 100%;
+        height: 300px;
     }
 
     .main-content .bottom-content {
         flex-wrap: wrap;
-        margin: 0 auto;
-        padding: 6% 0;
-        font-size: 1.6rem;
-        line-height: 1.8rem;
-    }
-
-    .main-content .bottom-content .text-content {
-        margin-bottom: 1rem;
+        align-content: space-between;
```

Comment:
Your fluid/responsive design is ah-mazing! I know that there was an issue with the cta image and text but I did say that you can ignore that. I hope you learned a lot from this project because it seems that it is really paying off in your work!